elastic/kibana
# [SecuritySolution] Fix KPIs disappear after clicking users / host risk score tabs

## Summary

Original issue: https://github.com/elastic/kibana/issues/145720
KPIs disappear after clicking users / host risk score tabs

Root cause:
useGlobalTime on users and host risk score tabs called `inputsActions.deleteAllQueries`, so KPIs' queries info were deleted accordingly. 

**Steps to Verify**

**Please enable feature flag:** `chartEmbeddablesEnabled`

1. Visit app/security/hosts/allHosts
2. Click on Host risk score tab and then click on anomalies tab
3. Please verify KPIs are displayed
4. Click any tabs in any order, please verify KPIs are displayed 

1. Visit app/security/users/allUsers
2. Click on User risk score tab and then click on anomalies tab
3. Please verify KPIs are displayed
4. Click any tabs in any order, please verify KPIs are displayed
